### PR TITLE
[layers] Update permute + nnstreamer layer to V2

### PR DESCRIPTION
**Pull request by kparichay**
Update Permute and NNStreamer Layers to LayerV2.

Signed-off-by: Parichay Kapoor <pk.kapoor@samsung.com>

### CODE REVIEW

**Review by lhs8928 — APPROVED** [review]
LGTM

File: nntrainer/layers/permute_layer.h [hunk 1]
```diff
@@ -76,48 +73,42 @@ class PermuteLayer : public LayerV1 {
...
-
-  using LayerV1::setProperty;
+  /**
+   * @copydoc Layer::supportBackwarding()
+   */
+  bool supportBackwarding() const { return true; }
 
   /**
-   * @copydoc Layer::setProperty(std::vector<std::string> values);
+   * @copydoc Layer::setProperty(const PropertyType type, const std::string
```
[the remark this answers is not in the document]
Comment (kparichay) [reply nntrainer/layers/permute_layer.h:106]: good catch, I will update.

**Review by jijoongmoon — APPROVED** [review]
LGTM

File: nntrainer/layers/permute_layer.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (kparichay) [reply nntrainer/layers/permute_layer.h]: done.

**Pull request merged**